src/control/client/pool.go

```diff
@@ -24,8 +24,8 @@
 package client
 
 import (
+	uuid "github.com/google/uuid"
```

Comment:
I assume this lib needs to be vendored too, right?

Reply:
weird didn't seem to be added even when specifying dep ensure -add, had to add it manually to the file but it was being pulled into vendor without it being in the file. maybe it was getting pulled in as a dependency, it's in there now anyway